runtime/lua/vim/_defaults.lua

```diff
@@ -215,173 +215,126 @@ do
 
   --- vim-unimpaired style mappings. See: https://github.com/tpope/vim-unimpaired
   do
+    ---@param lhs string
+    ---@param rhs function
+    ---@param desc string
+    local function create_unimpaired_mapping(lhs, rhs, desc)
```

Comment:
I think it might be better if `desc` is placed as the first or second argument as this will then make the `rhs` the last argument which is generally preferred for functions.

> - Any function signature that accepts a callback (example: |table.foreach()|)
  should place it as the LAST parameter (after opts), if possible (or ALWAYS
  for "continuation callbacks"—functions called exactly once).

Though this is only guidance, but I think it will help here.